sky/skypilot_config.py

```diff
@@ -219,14 +254,6 @@ def loaded_config_path() -> Optional[str]:
 _try_load_config()
 
 
-def _check_loaded_or_die():
```

Comment:
Is it ok to remove this?

Reply:
It is only called in `set_nested`. Doesn't seem reasonable to die to `set_nested` when global config is not loaded.